### PR TITLE
EOS-15358: Add ST for the fix in EOS-15332

### PR DESCRIPTION
**Pull request by DPG17**
- Added ST to validate issue fixed in EOS-15332.
 - Use a specific object key string that results into '/' and/or '+' in the final ListBucketResult's NextContinuationToken field/xml tag. During the run of ST, next continuation token was seen with '/', see below:
_<NextContinuationToken>GRoUERIQGyoBEAYBKkYtJDAtRyZDIjMwMTRBN0ZHND9GIUMzNCdBLSM7Pic0MEM/MCFHOSY/MzMjNDYgI0E4Ojg5NiY9QUcnLTMtR0c8LSwiLyZGLDMzIiwgPzBBJiE7PyUyMzE6OTwxOSEkMS9DP0ckOTQqQw==</NextContinuationToken>_ 

Signed-off-by: ‘Dattaprasad <dattaprasad.govekar@seagate.com>

### CODE REVIEW

File: st/clitests/awss3api_spec.py [hunk 1]
```diff
@@ -91,8 +91,42 @@ def create_object_list_file(file_name, obj_list=[], quiet_mode="false"):
...
+
+#  Create object list file
+object_list_file = create_object_list_file("obj_list_v2_keys.json", obj_list, "true")
+
+# List objects using list-objects-v2, page size 7 (first page will have 7 keys, and next page 3 keys)
+list_v2_options = {'page-size':7}
+result = AwsTest(('Aws list objects using page size \'%d\'') % (list_v2_options['page-size']))\
+    .list_objects_v2("seagatebucket", **list_v2_options)\
+    .execute_test()\
+    .command_is_successful()
```
Comment (amitwac2608) [inline st/clitests/awss3api_spec.py:119]: do we need to validate that we indeed got 7 objects on page 1 and remaining on page 2 ?

Comment (DPG17) [reply st/clitests/awss3api_spec.py:119]: not needed in this ST, as there are other existing STs (see line #337) that do that type of validation.

Comment (amitwac2608) [reply st/clitests/awss3api_spec.py:119]: ok, got it.

**Review by amitwac2608 — APPROVED** [review]

**Review by knrajnambiar76 — APPROVED** [review]

**Comment by DPG17** [comment]
Pre merge build http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/S3server-7.8.2003/197/ is triggered (in progress)

**Pull request merged**